config/jobs/kubernetes-sigs/cluster-api-provider-vsphere/cluster-api-provider-vsphere-presets.yaml

```diff
@@ -81,3 +81,40 @@ presets:
       - key: capv-gcs-keyfile.json
         path: keyfile.json
         mode: 288
+- labels:
+    preset-image-builder-e2e-config: "true"
+  env:
+  - name: GOVC_URL
+    valueFrom:
+      secretKeyRef:
+        name: cluster-api-provider-vsphere-ci
+        key: vmc-vcenter-url
+  - name: GOVC_USERNAME
+    valueFrom:
+      secretKeyRef:
+        name: cluster-api-provider-vsphere-ci
+        key: vmc-vcenter-user-image-builder
+  - name: GOVC_PASSWORD
+    valueFrom:
+      secretKeyRef:
+        name: cluster-api-provider-vsphere-ci
+        key: vmc-vcenter-password-image-builder
```

Comment:
I assume no VSPHERE_TLS_THUMBPRINT and VM_SSH_PUB_KEY is intended

Reply:
Yes, not used for image-builder.